src/pd/pd.rs

```diff
@@ -486,20 +591,36 @@ impl<T: PdClient> Runner<T> {
         self.is_hb_receiver_scheduled = true;
     }
 
+    fn handle_reconnect(&mut self) {
```

Comment:
This function does two things:
1. If `is_handle_reconnect_scheduled` is false, it means that the reconnect handler in pd client hasn't been initialized, we should set it.
2. If `is_handle_reconnect_scheduled` is true, it means that we have initialized the reconnect handler, then we should check if a reconnect has happened, and invalidate the cache if it did.

Am I right?

Reply:
Yes, I prefer that keep things together.

Reply:
But it is very confusing, it takes me a while to figure this.
I think you can just initialize the reconnect handler somewhere, and just handle reconnect here.

Reply:
I think we need comments here.